src/safari/iOS (Extension)/fr.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,9 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "Extension AEM Sidekick";
```

Comment:
I would rather say:
```suggestion
"CFBundleDisplayName" = "AEM Sidekick extension";
```

Are those translations managed by the translation team ? If yes, then current version is ok too.

Reply:
Yes they are